polaris.shopify.com/scripts/watch-md.mjs

```diff
@@ -0,0 +1,8 @@
+import path from 'path';
+import chokidar from 'chokidar';
+
+import genSiteJson from './gen-site-json.mjs';
+
+const mdPath = path.join(process.cwd(), 'content');
+
+chokidar.watch(mdPath).on('change', async () => await genSiteJson());
```

Comment:
This could probably be replaced with `chokidar-cli`. The downside is I would need a file that runs `genSiteJson()` which would look like...
```js
import genSiteJson from './gen-site-json.mjs';
genSiteJson();
```